README.md

```diff
@@ -0,0 +1,240 @@
+
+### Config mapping
+
+For more info: https://github.com/osbuild/image-builder-cli
+
+* **build_host.hostname** - IP or hostname with optional SSH port (required)
+* **build_host.username** - either root or username with sudo permissions (required)
+* **build_host.password** - SSH password when SSH keys are not available
+* **distro** - maps to `--distro`
+* **blueprint** - maps to `--blueprint`
+* **image_type** - maps to image type argument
+
+If there is an option missing, file an issue for us.
+
+## Building using bootc-image-builder
+
+Create a packer template named `template.pkr.hcl`:
+
+```
+packer {
+  required_plugins {
+    image-builder = {
+      source = "github.com/lzap/image-builder"
+      version = ">= 0.0.2"
+    }
+  }
+}
+
+source "image-builder" "example" {
+    build_host {
+        hostname = "buildhost.example.com"
+        username = "builder"
+    }
+
+    container_repository = "quay.io/centos-bootc/centos-bootc:stream9"
+
+    blueprint = <<BLUEPRINT
+[[customizations.user]]
+name = "user"
+password = "changeme"
+groups = ["wheel"]
+BLUEPRINT
+
+    image_type = "raw"
+}
+
+build {
+    sources = [ "source.image-builder.example" ]
+}
+```
+
+See [osbuild blueprint reference](https://osbuild.org/docs/user-guide/blueprint-reference/) for more info about blueprint format.
+
+Perform the build via:
+
+      packer init template.pkr.hcl
+      packer build template.pkr.hcl
+
+The image builder plugin will print last several lines from the image builder output as an artifact. To see more detailed output:
+
+      PACKER_LOG=1 packer build template.pkr.hcl
+
+### Config mapping
+
+For more info: https://github.com/osbuild/bootc-image-builder
+
+* **build_host.hostname** - IP or hostname with optional SSH port (required)
+* **build_host.username** - either root or username with sudo permissions (required)
+* **build_host.password** - SSH password when SSH keys are not available
+* **container_repository** - maps to container repository argument
+* **blueprint** - maps to `--blueprint`
+* **image_type** - maps to `--type`
+* **rootfs** - maps to `--rootfs`
+* **aws_upload.ami_name** - maps to AMI cloud uploader configuration
+* **aws_upload.s3_bucket** - maps to AMI cloud uploader configuration
+* **aws_upload.region** - maps to AMI cloud uploader configuration
+* **aws_upload.access_key_id** - maps to AMI cloud uploader configuration
+* **aws_upload.secret_access_key** - maps to AMI cloud uploader configuration
+
+If there is an option missing, file an issue for us.
+
+## Dry run
+
+If you want to perform, for any reason, a dry run where the main build command is `echo`ed to the console rather than executed, just set `IMAGE_BUILDER_DRY_RUN=1` environment variable when executing packer. Good for demos or testing the integration.
+
+## Building without Packer
+
+To test this library directly without packer, do:
+
+    go run github.com/lzap/packer-plugin-image-builder/cmd/ibpacker/ -help
```

Comment:
```suggestion
    go run github.com/osbuild/packer-plugin-image-builder/cmd/ibpacker/ -help
```

Reply:
Omg you could have just say "and there is more" I am doing it via grep/sed anyways, thanks :-)